packages/core/ui/TrackControls.js

```diff
@@ -8,39 +7,58 @@ import ToggleButton from '@material-ui/lab/ToggleButton'
 import { observer, PropTypes } from 'mobx-react'
 import ReactPropTypes from 'prop-types'
 import React from 'react'
-import buttonStyles from '../../LinearGenomeView/components/buttonStyles'
+import { getConf, readConfObject } from '../configuration'
+import { getSession } from '../util'
 
 const useStyles = makeStyles(theme => ({
   trackName: {
     margin: '0 auto',
     width: '90%',
-    fontSize: '80%',
+    fontSize: '0.8rem',
   },
-
   trackDescription: {
-    margin: '0.25em auto',
+    margin: '0 auto',
     width: '90%',
-    fontSize: '70%',
-    // color: '#5a5a5a',
+    fontSize: '0.7rem',
+  },
+  dragHandle: {
+    cursor: 'grab',
+    color: '#135560',
+  },
+  iconButton: {
+    padding: theme.spacing(0.5),
+  },
+  toggleButton: {
+    height: theme.spacing(4),
+    minWidth: 0,
+    border: 'none',
   },
-
-  ...buttonStyles(theme),
 }))
 
-function TrackControls({ track, view, onConfigureClick }) {
+function TrackControls({
+  track,
+  view,
+  onConfigureClick,
+  index,
+  children = null,
+  ...other
+}) {
   const classes = useStyles()
   let trackName = getConf(track, 'name') || track.id
   const session = getSession(track)
   if (getConf(track, 'type') === 'ReferenceSequenceTrack') {
     trackName = 'Reference Sequence'
-    session.datasets.forEach(datsetConf => {
-      const { assembly } = datsetConf
+    session.datasets.forEach(datasetConf => {
+      const { assembly } = datasetConf
       if (assembly.sequence === track.configuration)
         trackName = `Reference Sequence (${readConfObject(assembly, 'name')})`
     })
   }
   return (
-    <>
+    <div
+      data-testid={`trackControls-${view.id}-${getConf(track, 'configId')}`}
+      {...other}
```

Comment:
what is the other in this case?

Reply:
The styles and an "onDragEnter" handler. It's part of the peculiarity of the grid layout that there has to be an onDragEnter on both the track container and the track controls.

Reply:
would you mind adding that as a comment?